examples/all-clusters-app/all-clusters-common/gen/attribute-id.h

```diff
@@ -4467,6 +4467,16 @@
 #define ZCL_CLUSTER_REVISION_SERVER_ATTRIBUTE_ID (0xFFFD)
 #define ZCL_REPORTING_STATUS_SERVER_ATTRIBUTE_ID (0xFFFE)
 
+// Attribute ids for cluster: Binding
+
+// Client attributes
+#define ZCL_CLUSTER_REVISION_CLIENT_ATTRIBUTE_ID (0xFFFD)
+#define ZCL_REPORTING_STATUS_CLIENT_ATTRIBUTE_ID (0xFFFE)
+
+// Server attributes
+#define ZCL_CLUSTER_REVISION_SERVER_ATTRIBUTE_ID (0xFFFD)
+#define ZCL_REPORTING_STATUS_SERVER_ATTRIBUTE_ID (0xFFFE)
```

Comment:
So this is duplicating values for other clusters?

Could we just define the "global" attribute ids that are the same for all clusters once?  Followup for this, please.

Also, in CHIP, I don't see a "reporting status" attribute.  What I do see is a "FeatureMap" attribute with ID 0xFFFC.  Again, followup.

Reply:
> So this is duplicating values for other clusters?
> 
> Could we just define the "global" attribute ids that are the same for all clusters once? Followup for this, please.
> 
I opened #4219 for this one (with a patch).